UnitsNet.WindowsRuntimeComponent/GeneratedCode/Quantities/Acceleration.WindowsRuntimeComponent.g.cs

```diff
@@ -784,5 +784,107 @@ private static IFormatProvider GetFormatProviderFromCultureName([CanBeNull] stri
         {
             return cultureName != null ? new CultureInfo(cultureName) : (IFormatProvider)null;
         }
+
+        #region IConvertible Methods
+
+#if !WINDOWS_UWP
```

Comment:
Can't we skip/remove this entire section in WRC files? I think we have a `$wrc` variable in powershell you can use.

Reply:
Good call. Done!